.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint"],
+  "extends": ["plugin:@typescript-eslint/recommended"],
```

Comment:
can we use the same `eslintrc` across all projects? This way we'll have a common style across projects. Won't feel weird switching between repos.

For example syncing-server-js has https://github.com/standardnotes/syncing-server-js/blob/develop/.eslintrc

Reply:
Done 👍🏽